internal/dcache/replication_manager/replication_manager.go

```diff
@@ -491,6 +512,7 @@ retry:
 		rm.tp.schedule(&workitem{
 			targetNodeID: targetNodeID,
 			rvName:       rvName,
+			isLocalRV:    isLocalRV,
```

Comment:
also remove from here and let the the check be made right at the call site

Reply:
Done